lib/cloudant.js

```diff
@@ -712,7 +706,10 @@ Cloudant.prototype.updateAttributes = function(model, id, data, options, cb) {
     _.mergeWith(doc, data, function(dest, src) { return src; });
     self.create(model, doc, options, function(err) {
       if (err) return cb(err);
-      return cb(err, data);
+      mo.db.get(id, function(err, doc) {
```

Comment:
@rmg I talked with Sakib, connector function(in this case, `create`) only callback the `id` returned from database, then in juggler, we merge `id` and the data user provide in payload, then callback it to user. I bring up the question in slack channel, I am confused with the reason behind this design...